sacrebleu/tokenizers/tokenizer_ter.py

```diff
@@ -0,0 +1,148 @@
+import re
+
+from .tokenizer_none import NoneTokenizer
+
+
+def _normalize_general_and_western(sent: str) -> str:
+    # language-independent (general) part
+
+    # strip end-of-line hyphenation and join lines
+    sent = re.sub(r"\n-", "", sent)
+
+    # join lines
+    sent = re.sub(r"\n", " ", sent)
+
+    # handle XML escaped symbols
+    sent = re.sub(r"&quot;", "\"", sent)
```

Comment:
Ideally, this XML entities de-escaping should not be in the tokenizers, but in reading references (https://github.com/mjpost/sacrebleu/blob/e22640c996ca48d018fd974ba26a052180f4c77d/sacrebleu/utils.py#L53) (system outputs are provided already as plaintext to SacreBLEU, so the de-escaping should *not* be applied on these).
However, the same bug (#15) is in BLEU's default tokenizer [13a](https://github.com/mjpost/sacrebleu/blob/master/sacrebleu/tokenizers/tokenizer_13a.py), so I suggest to merge this PR as is and fix this later.

Reply:
Thanks for the suggestion. Would it make sense to simply remove these lines from the tokenizer then?

Note though that by default, this normalization is disabled, so it won't be applied unless the user configures it (which can't be done through command-line options for the time being anyway).

Also, if someone was looking (for some obscure reason) for a Python tokenizer that's consistent with Tercom, they could simply use the class `TercomTokenizer` and get the desired result. If we removed this normalization, the tokenizer would no longer be exactly the same.